main.py

```diff
@@ -252,36 +255,13 @@ def get_board(self, access_token_in):
             )
         )
         ws.recv()
-        ws.send(
-            json.dumps(
-                {
-                    "id": "2",
-                    "type": "start",
-                    "payload": {
-                        "variables": {
-                            "input": {
-                                "channel": {
-                                    "teamOwner": "AFD2022",
-                                    "category": "CANVAS",
-                                    "tag": "0",
-                                }
-                            }
-                        },
-                        "extensions": {},
-                        "operationName": "replace",
-                        "query": "subscription replace($input: SubscribeInput!) {\n  subscribe(input: $input) {\n    id\n    ... on BasicMessage {\n      data {\n        __typename\n        ... on FullFrameMessageData {\n          __typename\n          name\n          timestamp\n        }\n        ... on DiffFrameMessageData {\n          __typename\n          name\n          currentTimestamp\n          previousTimestamp\n        }\n      }\n      __typename\n    }\n    __typename\n  }\n}\n",
-                    },
-                }
-            )
-        )
-        ws.recv()
 
         image_sizex = 2
         image_sizey = 1
 
         imgs = []
         already_added = []
-        for i in range(1, image_sizex * image_sizey + 1):
+        for i in range(0, image_sizex * image_sizey):
```

Comment:
What does that do exactly ?

Reply:
i missed that this loop was changed to fix the problem of getting the first canvas image twice, this undoes that change so the hard coded first request could still be removed.

Reply:
Ok I think I get it. So now everything works ?

Reply:
yes. i have been running this version and everything seems to work

Reply:
Ok, can you fix the conflicts so this can be merged ?